### PR TITLE
Add Headers Lincense to the project code base

### LINKED ISSUE #4735: All Lincense Headers should be on top of package not inside package
### Feature Request

I am proposing this change in our code base, All linces Headers should come on top of package, this is to fix is really very small but it brings the aspect of clean code. Let me hear from your suggestions.

### PR DESCRIPTION
**Pull request by sherrif10**
Fixes #4735

Proposed Changes. 
As fur as Licensing is concerned, all the classes are supposed to have linces and some classes didnot have linces so we needed to have all the classes lincesed.

Changes proposed in this pull request:
- Added Linces Headers to the classes that had no linces Headers. cc @antoine2711  @WaltonG .Kindly review

### CODE REVIEW

**Comment by WaltonG** [comment]
> Fixes #{issue number[ here](https://github.com/OpenRefine/OpenRefine/issues/4735)}
> 
> Added Linces Headers to the classes that had no linces Headers. cc @antoine2711 @WaltonG .Kindly review
> ## Changes proposed in this pull request:

Include the issue number for Github to link the  PR with the issue. It would also be great to include the proposed changes

**Comment by sherrif10** [comment]
Here is the issue https://github.com/OpenRefine/OpenRefine/issues/4735

File: server/IDEs/eclipse/Refine with extensions.launch [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+    <booleanAttribute key="org.eclipse.jdt.launching.ATTR_USE_CLASSPATH_ONLY_JAR" value="false"/>
+    <stringAttribute key="org.eclipse.jdt.launching.CLASSPATH_PROVIDER" value="org.eclipse.m2e.launchconfig.classpathProvider"/>
+    <stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="com.google.refine.Refine"/>
+    <stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="server"/>
+    <stringAttribute key="org.eclipse.jdt.launching.SOURCE_PATH_PROVIDER" value="org.eclipse.m2e.launchconfig.sourcepathProvider"/>
+    <stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-Drefine.autoreloading=true -Dbutterfly.modules.path=modules,&quot;${workspace_loc:extensions}&quot;,&quot;${workspace_loc:jython}&quot;,&quot;${workspace_loc:extensions}&quot;,&quot;${workspace_loc:database}&quot;"/>
+    <stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="${workspace_loc:OpenRefine}"/>
```
Comment (wetneb) [inline server/IDEs/eclipse/Refine with extensions.launch:17]: Those look like unrelated change, right?

File: main/src/com/google/refine/util/HttpClient.java [hunk 2]
```diff
@@ -1,3 +1,35 @@
+/*
+
+Copyright 2010, Google Inc.
```
Comment (wetneb) [inline main/src/com/google/refine/util/HttpClient.java:3]: We probably should not "Google Inc." as the copyright holder, but rather "OpenRefine contributors"

Comment (sherrif10) [reply main/src/com/google/refine/util/HttpClient.java:3]: okay thanks let me correct that.

File: server/IDEs/eclipse/Refine with extensions.launch (hunk 1 above)
Comment (sherrif10) [reply server/IDEs/eclipse/Refine with extensions.launch:17]: Yes @wetneb , these are un related and they got no effect.

**Comment by antoine2711** [comment]
> Here is the issue #4735

Look at the change I made to the description of the PR.
It must starts like this:

Fixes #4735

"Fixes" + SPACE + # + Issue number.

Regards, Antoine

File: main/src/com/google/refine/util/HttpClient.java (hunk 2 above)
Comment (wetneb) [reply main/src/com/google/refine/util/HttpClient.java:3]: No that would not be appropriate. We should only add the "OpenRefine contributors" header to the files which do not have a header yet. The existing headers should be left as they are.

**Comment by sherrif10** [comment]
Thanks @antoine2711

File: main/src/com/google/refine/util/HttpClient.java (hunk 2 above)
Comment (sherrif10) [reply main/src/com/google/refine/util/HttpClient.java:3]: Point noted thanks  Let me update it .

**Comment by sherrif10** [comment]
Hello @wetneb  updated the linces Headers

File: server/IDEs/eclipse/Refine with extensions.launch (hunk 1 above)
Comment (wetneb) [reply server/IDEs/eclipse/Refine with extensions.launch:17]: Then we should make sure this pull request leaves this file untouched.

**Comment by sherrif10** [comment]
Hello @wetneb  am opening a new pull request as the changes failed to be undo.

**Review by wetneb — APPROVED** [review]
